rdrf/rdrf/forms/components.py

```diff
@@ -135,10 +135,19 @@ def _get_template_data(self):
             "consent_locked": self.consent_locked,
             "clinician_form_link": self._get_clinician_form_link(),
             "proms_link": self._get_proms_link(),
+            "allow_proms_add": self._get_proms_add(),
         }
 
         return data
 
+    def _get_proms_add(self):
+        metadata = self.registry_model.metadata
+        allow_proms_add = metadata.get("allow_proms_add", None)
+        if allow_proms_add is None:
+            return ""
```

Comment:
Would make more sense for this to return False - returning empty string relies on python's truthyness evaluation rules and will work but why not return an explicit Boolean?